minecraft/bukkit/src/main/java/dev/triumphteam/cmd/bukkit/BukkitCommand.java

```diff
@@ -37,10 +37,7 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
This isn't that big of a deal as I can fix it late, but avoid wildcard imports.
You can leave it though so dw about it right now. ;p

Reply:
Sorry, my IDE seems to do this automatically. One of these days I'll get around to disabling it